roles/collect/tasks/main.yml

```diff
@@ -76,12 +65,17 @@
 # this appends the string inside the brackets to the 'api_urls' list.
 - name: assemble compiled URL facts, append to list.
   set_fact:
-    api_urls: "{{ api_urls + [api_host+'/'+api_path+item] }}"
+    api_urls: "{{ api_urls + [api_path+item] }}"
   with_items: "{{ api_params }}"
 
+# Use a cluster specific path
+- name: Set download_path
+  set_fact:
+    temp_download_path: "{{ collect_download_path }}/{{ ocp_config['ocp_cluster_id'] }}/korekuta_collect"
```

Comment:
Can we preserve the original behavior where all files korekuta works with are collected into a single directory, please?

Unless there's some technical reason to change the behavior, I'd prefer this instead:
``` 
collect_download_path: '/tmp/korekuta-collect'
[...]
temp_download_path: "{{ collect_download_path }}/{{ ocp_config['ocp_cluster_id'] }}"
```

Reply:
There is a reason. We have multiple users setup on the same machine, each uses korekuta to access a different cluster. The first user to run this owns the korekuta-collect directory and blocks other users from using it. Would you be more comfortable with setting global write permissions to the directory?

Reply:
Sure. 0777 permissions is fine. 

We may need to modify the clean-up code path, otherwise there is a race condition there.